ee/modules/110-istio/docs/EXAMPLES.md

```diff
@@ -188,7 +188,83 @@ spec:
       weight: 10
 ```
 
-## Ingress
+## Ingress to publish applications
+
+### Istio Ingress Gateway
+
+Exmaple:
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: IngressIstioController
+metadata:
+ name: main
+spec:
+  # ingressGatewayClass contains the label selector value used to create the Gateway resource
+  ingressGatewayClass: istio-hp
+  inlet: HostPort
+  hostPort:
+    httpPort: 80
+    httpsPort: 443
+  nodeSelector:
+    node-role/frontend: ''
+  tolerations:
+    - effect: NoExecute
+      key: dedicated
+      operator: Equal
+      value: frontend
+  resourcesRequests:
+    mode: VPA
+```
+
+```yaml
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway-app
+spec:
+  selector:
+    # label selector for using the Istio Ingress Gateway main-hp
+    istio.deckhouse.io/ingress-gateway-class: istio-hp 
+  servers:
+    - port:
+        # standard template for using the HTTP protocol
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
+        - app.example.com
+    - port:
+        # standard template for using the HTTPS protocol
+        number: 443
+        name: https
+        protocol: HTTPS
+      tls:
+        mode: SIMPLE
+        # a secret with a certificate and a key, which must be created in the d8-istio-ingress namespace
+        # supported secret formats can be found at https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#key-formats
+        credentialName: app-tls-secrets
+      hosts:
+        - app.example.com
+```
+
```

Comment:
add a Secret example.

Reply:
And we need to discuss how to use Certificate's to order certs from cert-manager.

Reply:
secret has been added